src/main/java/edu/ucsb/cs156/rec/entities/RequestType.java

```diff
@@ -0,0 +1,27 @@
+package edu.ucsb.cs156.rec.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * This is a JPA entity that represents a RequestType
+ */
```

Comment:
Might be good to include a few examples, since this comment will be in the Javadoc for this class.  That will help future developers understand what this is for.  You can borrow from the text in the original issues.

Reply:
We added the examples.